### PR TITLE
[Add] Remove the default price list in quotation/sale

### PR DESCRIPTION
**Pull request by ehdem**
In quotaion/sale form view, display the configured price list
for the selected partner

### CODE REVIEW

File: sale_quotation_remove_default_pricelist/sale.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+##############################################################################
+
+from openerp.osv import fields, orm
+
+
+class sale(orm.Model):
+    _inherit = 'sale.order'
+
+    def onchange_shop_id(self, cr, uid, ids, shop_id, context=None):
```
Comment (joaoalf) [inline sale_quotation_remove_default_pricelist/sale.py:29]: @ehdem It's better to call the super to not break the inherit chain of the method and then tweak the result.

**Comment by joaoalf** [comment]
@ehdem I'm not sure that sale-report is the best place to hold this module.

File: sale_quotation_remove_default_pricelist/sale.py (hunk 1 above)
Comment (ehdem) [inline sale_quotation_remove_default_pricelist/sale.py:29]: @joaoalf : yep

**Comment by ehdem** [comment]
@joaoalf: What is the best place for this new module ?

**Comment by joaoalf** [comment]
@ehdem Maybe sale-workflow.

**Comment by ehdem** [comment]
@joaoalf: thanks you, let's do it

**Comment by ehdem** [comment]
This PR is moved towards sale-workflow. New PR: https://github.com/OCA/sale-workflow/pull/131

**Pull request closed without merging**